js/components/volumeControls.js

```diff
@@ -8,6 +8,15 @@ var CONSTANTS = require('../constants/constants');
 
 var VolumeControls = React.createClass({
 
+  volumeChange: function(vol) {
+    var newVol = Utils.ensureNumber(vol);
```

Comment:
I think this actually returns `null` when it's not a valid number, but you can pass a default value to return for these cases in the second parameter

Reply:
Done.